.eslintignore

```diff
@@ -0,0 +1,2 @@
+eslint-disable react/prop-types: 0
+eslint-disable react/destructuring-assignment: 0
```

Comment:
Please kindly remove these two lines and try to fix the error using react prop-types. You can find more information about how to fix this [here](https://reactjs.org/docs/typechecking-with-proptypes.html)